exampleFirstEntity/1.createEntityAlastriaID.js

```diff
@@ -16,7 +16,7 @@ const keyDataEntity1 = fs.readFileSync(
 )
 const keystoreDataEntity1 = JSON.parse(keyDataEntity1)
 const keyDataAdmin = fs.readFileSync(
```

Comment:
I think it is the time to change some variables in this file. "admin" no longer exists. Lets call it "firstIdentity". Please check the other files in `exampleFirstEntity`

Reply:
I refactored the code, take a look if you can to make sure the changes to some README files are correct too. I also tested the examples after the changes and looks like everything is working as it should.